src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="index.html" method="get">
+      <div class="form-big" data-qa="big">
+        <label class="
+        form__loop
+        form__loop--size_big
+        form-big__big-loop"
+        for="big-loop"
+        >
+        </label>
+        <input class="
+        form form--input_big
+        form-big__form"
+        type="text"
+        name="big-loop"
+        id="big-loop"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        >
```

Comment:
check it in other places

```suggestion
        <input 
          class=" form form--input_big form-big__form"
          type="text"
          name="big-loop"
          id="big-loop"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
```